stable/concourse/README.md

```diff
@@ -234,6 +236,15 @@ The following table lists the configurable parameters of the Concourse chart and
 | `secrets.oauthAuth.groupsKey` | OAuth2 Authentication: The groups key indicates which claim to use to map external groups to Concourse teams. | `nil` |
 | `secrets.oauthAuth.caCert` | OAuth2 Authentication: CA Certificate | `nil` |
 | `secrets.oauthAuth.skipSslValidation` | OAuth2 Authentication: Skip SSL validation | `nil` |
+| `secrets.oidcAuth.enabled` | OIDC Authentication: (Required) Enable OIDC authentication | `false` |
+| `secrets.oidcAuth.displayName` | OIDC Authentication: The auth provider name displayed to users on the login page | `nil` |
+| `secrets.oidcAuth.issuer` | OIDC Authentication: (Required if OIDC enabled) An OIDC issuer URL that will be used to discover provider configuration using the .well-known/openid-configuration | `nil` |
+| `secrets.oidcAuth.clientId` | OIDC Authentication: (Required if OIDC enabled) Client id | `nil` |
+| `secrets.oidcAuth.clientSecret` | OIDC Authentication: (Required if OIDC enabled) Client secret | `nil` |
+| `secrets.oidcAuth.scope` | OIDC Authentication: Any additional scopes that need to be requested during authorization | `nil` |
+| `secrets.oidcAuth.groupsKey` | OIDC Authentication: The groups key indicates which claim to use to map external groups to Concourse teams. | `nil` |
```

Comment:
Would it be better to avoid adding this parameter all together? Turns out OpenID provider in coreos/dex doesn't support group claims: https://github.com/coreos/dex. We integrated with Azure, but there is to way to retrieve the list of groups user belongs to. So this parameter doesn't seem to have any effect on the behavior.

Reply:
I haven't actually used OIDC Authentication at all, so I'm not sure—I've just folded in all of the flags Concourse supports. I recommend asking the Concourse team directly, probably at https://discuss.concourse-ci.org/ .

Reply:
I filed https://github.com/concourse/concourse/issues/2538, let's see what the response is